### PR TITLE
DOCS-6697 removed requirement for alphanumeric characters

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/layers.adoc [hunk 1]
```diff
@@ -23,7 +23,6 @@ This is an application property, not a server property.
 When this property is used, nodes automatically appear in the specified layer. If the layer doesn't exist it is created.
 
 [NOTE]
```
Comment (sarathecoubian) [inline modules/ROOT/pages/layers.adoc:22]: @asimons-mule We have another note very close to this one. Are we able to combine these or remove a Note designation?

Comment (asimons-mule) [reply modules/ROOT/pages/layers.adoc:22]: Good point. The first note: "This is an application property, not a server property" can just become the final sentence of the paragraph above it.

**Review by asimons-mule — APPROVED** [review]
Looks good!

File: modules/ROOT/pages/customize-your-canvas.adoc [hunk 2]
```diff
@@ -15,8 +15,7 @@ Visualizer editors can save defined views so that any Visualizer user can access
 When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.
 Layers, Tags and Display Names can all be customized in the Visualizer UI or by using a runtime property.
```
Comment (kbostrom) [inline modules/ROOT/pages/customize-your-canvas.adoc]: Active voice
```suggestion
You can use international letters, numbers, and symbols to customize Layers, Tags, and Display Names values in the Visualizer UI or by using a runtime property.
```

Comment (kbostrom) [reply modules/ROOT/pages/customize-your-canvas.adoc]: Suggest the previous rewording to omit the text that follows.

File: modules/ROOT/pages/customize-your-canvas.adoc [hunk 3]
```diff
@@ -15,8 +15,7 @@ Visualizer editors can save defined views so that any Visualizer user can access
...
 
-Note that the runtime property supports only alphanumeric characters and hyphens.
-Special characters and spaces will be replaced with a comma.
+Anypoint Visualizer supports the use of international characters when you set layer names, tag names, and display names, whether you set them in the UI or using a runtime property.
```
Comment (kbostrom) [inline modules/ROOT/pages/customize-your-canvas.adoc]: omit

File: modules/ROOT/pages/layers.adoc [hunk 4]
```diff
@@ -15,15 +15,11 @@ Map a node to a layer in the application's properties file, or use the Anypoint
...
 === Set a Layer with Properties
 
-To automate the classification of nodes into layers, configure the property `anypoint.platform.visualizer.layer=<name>` in the application's properties file.
+To automate the classification of nodes into layers, configure the property `anypoint.platform.visualizer.layer=<name>` in the application's properties file. Note that this is an application property, not a server property.
 
-[NOTE]
-This is an application property, not a server property.
-
-When this property is used, nodes automatically appear in the specified layer. If the layer doesn't exist it is created.
+When this property is used, nodes automatically appear in the specified layer. If the layer doesn't exist, it is created.
```
Comment (kbostrom) [inline modules/ROOT/pages/layers.adoc]: active
```suggestion
When you use this property, nodes automatically appear in the specified layer. If the layer doesn't exist, it is created.
```

File: modules/ROOT/pages/use-tags-in-visualizer.adoc [hunk 5]
```diff
@@ -15,9 +15,6 @@ Note that this is an application property, not a server property.
 
 When this property is used, the nodes automatically show the assigned tag(s) in the *View* card of Visualizer. The tag cannot be overridden in the UI and can only be modified in properties. You can still add other tags to this node using the UI, however. All tags are displayed as lowercase.
```
Comment (kbostrom) [inline modules/ROOT/pages/use-tags-in-visualizer.adoc]: active
```suggestion
When you use this property, the nodes automatically show the assigned tags in the *View* card of Visualizer. You can add but cannot override tags in the UI; you can modify them only in properties. All letters in tag names are displayed in lowercase.
```

**Review by kbostrom — APPROVED** [review]
Done!

File: modules/ROOT/pages/customize-your-canvas.adoc (hunk 3 above)
Comment (sarathecoubian) [inline modules/ROOT/pages/customize-your-canvas.adoc]:
```suggestion
Anypoint Visualizer supports all unicode characters with UTF-8 encoding, including international characters, when you set layer names, tag names, and display names, whether you set them in the UI or using a runtime property.
```

File: modules/ROOT/pages/customize-your-canvas.adoc [hunk 6]
```diff
@@ -15,8 +15,7 @@ Visualizer editors can save defined views so that any Visualizer user can access
 When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.
 Layers, Tags and Display Names can all be customized in the Visualizer UI or by using a runtime property.
 
-Note that the runtime property supports only alphanumeric characters and hyphens.
-Special characters and spaces will be replaced with a comma.
+Anypoint Visualizer supports all unicode characters with UTF-8 encoding, including international characters, when you set layer names, tag names, and display names, whether you set them in the UI or using a runtime property.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/customize-your-canvas.adoc]:
```suggestion

```

File: modules/ROOT/pages/customize-your-canvas.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/customize-your-canvas.adoc]:
```suggestion
You can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers) to customize Layers, Tags, and Display Names in the Visualizer UI or by using a runtime property.
```

File: modules/ROOT/pages/customize-your-canvas.adoc [hunk 7]
```diff
@@ -13,10 +13,8 @@ The *View* card and the canvas are updated to display the current selection.
 
 Visualizer editors can save defined views so that any Visualizer user can access them.
 When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.
-Layers, Tags and Display Names can all be customized in the Visualizer UI or by using a runtime property.
+You can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers) to customize Layers, Tags, and Display Names in the Visualizer UI or by using a runtime property.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/customize-your-canvas.adoc]:
```suggestion
If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers) to customize Layers, Tags, and Display Names in the Visualizer UI or by using a runtime property.
```

**Pull request merged**